examples/imshow-xavier/readme.md

```diff
@@ -0,0 +1,27 @@
+# Imshow jetson example
```

Comment:
'jetson' word needs to be replaced with 'xavier', right?

Reply:
fixed